src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/FilterVariantTranches.java

```diff
@@ -133,8 +139,8 @@ public void firstPassApply(final VariantContext variant, final ReadsContext read
 
     @Override
     public void afterFirstPass() {
-        logger.info(String.format("Found %d SNPs %d indels with INFO score key:%s.", scoredSnps, scoredIndels, infoKey));
-        logger.info(String.format("Found %d SNPs %d indels in the resources.", snpScores.size(), indelScores.size()));
+        logger.info(String.format("Found %d SNPs and %d indels with INFO score key:%s.", scoredSnps, scoredIndels, infoKey));
+        logger.info(String.format("Found %d SNPs and %d indels in the resources.", snpScores.size(), indelScores.size()));
 
         if (scoredSnps == 0 || scoredIndels == 0 || snpScores.size() == 0 || indelScores.size() == 0){
             throw new GATKException("VCF must contain SNPs and indels with scores and resources must contain matching SNPs and indels.");
```

Comment:
This should be a UserException. GATKException is for unexpected errors in the code that are outside the user’s control, like the one thrown by filterStringFromScore.

Reply:
fixed